wgengine/router/router_linux.go

```diff
@@ -365,7 +365,13 @@ func (r *linuxRouter) setNetfilterMode(mode NetfilterMode) error {
 // addAddress adds an IP/mask to the tunnel interface. Fails if the
 // address is already assigned to the interface, or if the addition
 // fails.
-func (r *linuxRouter) addAddress(addr netaddr.IPPrefix) error {
+func (r *linuxRouter) addAddress(addr netaddr.IPPrefix) (ret error) {
```

Comment:
Confused by this, I thought we weren't even trying to add IPv6 at all, so the fact that it gets all the way to invoking iproute2 is the bug, afaict.

Will look during a gap in onsite today.